doc/source/faq.rst

```diff
@@ -0,0 +1,113 @@
+and Mathematica. While the exact function names differ a bit, most functionality
+is shared, so if you can code any of them you can use |igraph|: just refer to
+the installation instructions for the appropriate language on our
+`homepage <https://igraph.org>`_.
+
+If you are not familiar with programming at all, or if you don't know any Python
+but would still like to use the Python interface for |igraph|, you should start by
+learning Python first. There are many resources online including online classes,
+videos, tutorials, etc. |igraph| does not use a lot of advanced Python-specific
+tricks, so once you can use a standard module such as :mod:`pandas` or
+:mod:`matplotlib`, |igraph| should be easy to pick up.
+
+
+I would like to have a function for operation/algorithm X, can you add it?
+--------------------------------------------------------------------------
+We are continuously extending |igraph| to include new functionality, and
+requests from our community are the best way to guide those efforts. Of
+course, we are just a few folks so we cannot guarantee that each and every
+obscure community detection algorithm will be included in the package.
+Please open a new thread on our `forum <https://igraph.discourse.group/>`_
+describing your request. If your request is to adapt an existing function
+or specific piece of code, you can directly open a
+`GitHub issue <https://github.com/igraph/python-igraph/issues>`_
+(make sure a similar issue does not exist yet! - If it does, comment there
+instead.)
+
+
+What's the difference between |igraph| and similar packages (networkx, graph-tool)?
+-----------------------------------------------------------------------------------
+All those packages focus on graph/network analysis.
+
+**Differences:**
+
+ - |igraph| supports **multiple programming languages** (e.g. C, Python, R, Mathematica). `networkx`_ and `graph-tool`_ are Python only.
+ - |igraph|'s core library is written in C, which makes it often faster than `networkx`_. `graph-tool`_ is written in heavily templated C++, so it can be as fast as |igraph| but supports fewer architectures. Compiling `graph-tool` can take much longer than |igraph| (hours versus around a minute).
+ - |igraph| vertices are *ordered with contiguous numerical IDs, from 0 upwards*, and an *optional* "vertex name". `networkx`_ nodes are *defined* by their name and not ordered.
+ - Same holds for edges, ordered with integer IDs in |igraph|, not so in `networkx`_.
+ - |igraph| can plot graphs using :mod:`matplotlib` and has experimental support for `plotly`_, so it can produce animations, notebook widgets, and interactive plots (e.g. zoom, panning). `networkx`_ has excellent :mod:`matplotlib` support but no `plotly`_ support. `graph-tool`_ only supports static images via Cairo and GTK+.
+ - |igraph| and `networkx`_ are maintained by teams of developers. `graph-tool`_ is developed by a single (very active) person.
```

Comment:
I'd remove this item in case it changes in the future; we will probably not notice it and will forget to update the comparison.

Also, for this section, I'd probably add a sentence stating that these were true at the time of writing (i.e. Jan 2022), but if anyone finds out that the situation has changed, let us know and/or file a PR.

Reply:
yep